tgui/src/interfaces/rpd/dirsel.ract

```diff
@@ -3,7 +3,7 @@
 		<ui-section>
 			{{#each previews}}
 				<ui-button state='{{selected ? "selected" : null}}' action='setdir'
-					params='{"dir": {{dir}}, "flipped": {{flipped}}}'><img src='pipe.{{dir}}.{{icon_state}}.png' title='{{dir_name}}'/></ui-button>
+					params='{"dir": {{dir}}, "flipped": {{flipped}}}'><span class='pipes32x32 {{dir}}-{{icon_state}}' title='{{dir_name}}'></span></ui-button>
```

Comment:
Could probably do with a helper somewhere in base tgui files.

Reply:
Maybe so, not familiar enough with react to make it actually usable though.

Reply:
Fair enough, this is not very important.